sdk/data/aztables/README.md

```diff
@@ -39,11 +39,21 @@ az storage account show -n mystorageaccount -g MyResourceGroup --query "primaryE
 ```
 
 Once you have the account URL, it can be used to create the service client:
-```golang
-cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
-handle(err)
-serviceClient, err := aztables.NewServiceClientWithSharedKeyCredential("https://<myAccountName>.table.core.windows.net/", cred, nil)
-handle(err)
+```go
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/data/aztables"
+)
+
+func main() {
+	cred, err := aztables.NewSharedKeyCredential("<myAccountName>", "<myAccountKey>")
```

Comment:
Just a nitpick that sometimes you include the angle brackets for the string placeholder on this method and sometimes you leave it out. Maybe we should adopt a standard and try to be consistent throughout our READMEs/samples.